docs/api/public_php_api_managing_content.md

```diff
@@ -201,3 +201,57 @@ $copyUpdateStruct->identifier = $newIdentifier;
 $this->contentTypeService->updateContentTypeDraft($copyDraft, $copyUpdateStruct);
 $this->contentTypeService->publishContentTypeDraft($copyDraft);
 ```
+
+## Calendar events
+
+### Getting events
+
+To get a list of all events for a specified time period, use the `CalendarService::getEvents` method.
+You need to provide the method with an EventQuery, which takes a date range and a count as the minimum of parameters:
+
+``` php
+$dateFrom = new \DateTimeImmutable('2020-08-01T10:00:00+00:00');
+$dateTo = new \DateTimeImmutable('2020-10-31T10:0:00+00:00');
+$dateRange = new Calendar\DateRange($dateFrom, $dateTo);
+
+$eventQuery = new Calendar\EventQuery($dateRange, 10);
+
+$eventList = $this->calendarService->getEvents($eventQuery);
+
+foreach ($eventList as $event) {
+    $output->writeln($event->getName() . '; date: ' . $event->getDateTime()->format('T Y-m-d H:i:s') );
+}
+```
+
+You can also get the first and last event in the list by using the `first()` and `last()` methods of an `EventCollection` respectively:
+
+``` php
+$eventCollection = $eventList->getEvents();
+$output->writeln('First event: ' . $eventCollection->first()->getName() . '; date: ' . $eventCollection->first()->getDateTime()->format('T Y-m-d H:i:s') );
+```
+
+You can modify the events in a collection using the `find(Closure $predicate)`, `filter(Closure $predicate)`,
+`map(Closure $callback)` or `slice(int $offset, ?int $length = null)` methods of `EventCollection`, for example:
```

Comment:
> You can modify the events in a collection using the ...

From the technical POV this statement is "not true". But you can rephrase it to

> You can process the events in a collection using the...

and will be fine.